webclient/src/containers/Login/Login.tsx

```diff
@@ -73,13 +75,63 @@ const Login = ({ state, description }: LoginProps) => {
     closeResetPasswordDialog();
   }, ServerTypes.RESET_PASSWORD_SUCCESS, []);
 
+  useReduxEffect(({ options: { hashedPassword } }) => {
+    if (hostIdToRemember) {
+      HostDTO.get(hostIdToRemember).then(host => {
+        host.hashedPassword = hashedPassword;
+        host.save();
+      });
+    }
+  }, ServerTypes.LOGIN_SUCCESSFUL, [hostIdToRemember]);
+
   const showDescription = () => {
     return !isConnected && description?.length;
   };
 
-  const createAccount = () => {
+  function createAccount() {
     console.log('Login.createAccount->openForgotPasswordDialog');
-  };
+  }
+
+  const onSubmit = useCallback((loginForm) => {
+    const {
+      userName,
+      password,
+      selectedHost,
+      selectedHost: {
+        id: hostId,
+        hashedPassword
+      },
+      remember
+    } = loginForm;
+
+    updateHost(loginForm);
+
+    if (remember) {
+      setHostIdToRemember(hostId);
+    }
+
+    const options: WebSocketConnectOptions = {
+      ...getHostPort(selectedHost),
+      userName,
+      password
+    };
+
+    if (!password) {
+      options.hashedPassword = hashedPassword;
+    }
+
+    AuthenticationService.login(options as WebSocketConnectOptions);
```

Comment:
will this drop the extra field(s) (like hashedPassword) before logging in?

Reply:
If passwordSalt is supported, the protobuf command will look for a hashedPassword, or hash the password itself if hashedPassword wasnt provided. If its not supposed, it will just use the naked password.